### DISCUSSION TITLE
Does the `package.json#scripts.dependencies` lifecycle script work? If so, how/when?

### DISCUSSION THREAD

**Discussion by broofa** [npm/feedback#843] [General]
In the NPM  ["Life Cycle Scripts" docs](https://docs.npmjs.com/cli/v9/using-npm/scripts#life-cycle-scripts) there's mention of a "`dependencies`" script that "runs AFTER any operations that modify the node_modules directory IF changes occurred."

Is this actually a thing?  Does it work?  If so, how?  In looking through the 2,000+ package.json files that sit under `node_modules` in our monorepo, I don't find any that use this hook.  Nor have I been able to get it to work in my local testing.

For example, it sounds like I should be able to create a module that has a peer dependency on `mime-db` like this ...
```jsonc
// package.json
{
  "peerDependencies": {
    "mime-db": "^1.0.0"
  },
  "scripts": {
    // ...
    "dependencies": "echo \"DEPENDENCIES\""
  }
}
```
... and install it as part of some project.  Then [in the project] run `npm install mime-db@1.45.0`" to switch to a different `mime-db` version, and see "DEPENDENCIES" show up somwhere in the console (because the `dependencies` hook was run).  But that doesn't happen.   Should it?  If so, what am I doing wrong?

**Comment by ljharb** [comment]
What version of npm are you using? (I'd expect that the docs would either match v6 latest, or v9 latest)

**Reply by broofa** [reply]
@ljharb 
```bash
$ npm --version
8.19.2
```

**Comment by broofa** [comment]
FWIW, I updated to `npm@latest` (9.2.0) and still doesn't seem like it's working.

[Edit to add:  Looking at the different docs versions, it looks like support for this was added in npm@8]
